ports/esp32/boards/WAVESHARE_S3_ZERO/mpconfigboard.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MICROPY_HW_BOARD_NAME
+// Can be set by mpconfigboard.cmake.
+#define MICROPY_HW_BOARD_NAME               "ESP32-S3-Zero module"
+#endif
+#define MICROPY_HW_MCU_NAME                 "ESP32S3"
+
+#define MICROPY_PY_MACHINE_DAC              (0)
+
+// Enable UART REPL for modules that have an external USB-UART and don't use native USB.
+#define MICROPY_HW_ENABLE_UART_REPL         (1)
+
+#define MICROPY_HW_I2C0_SCL                 (9)
+#define MICROPY_HW_I2C0_SDA                 (8)
```

Comment:
Was there a reason to specify these? This board doesn't seem to explicitly define them for I2C.